lib/Elastica/Bulk/Action/AbstractDocument.php

```diff
@@ -9,20 +9,20 @@
 abstract class AbstractDocument extends Action
 {
     /**
-     * @var \Elastica\Document|\Elastica\Script\AbstractScript
+     * @var Document|AbstractScript
      */
     protected $_data;
 
     /**
-     * @param \Elastica\Document|\Elastica\Script\AbstractScript $document
+     * @param Document|AbstractScript $document
```

Comment:
The question probably is, if phpdoc automatically also detects this. So I assume the answer is yes.